packages/types-algosdk/types/index.d.ts

```diff
@@ -435,7 +435,14 @@ declare module 'algosdk' {
 
   export function signBid(bid: any, sk: any): any;
 
-  export function signBytes(bytes: any, sk: any): any;
+  /**
+   * signBytes takes arbitrary bytes and a secret key, prepends the bytes with "MX" for domain separation, signs the bytes 
+   * with the private key, and returns the signature.
+   * @param bytes Uint8array
```

Comment:
We don't need to repeat the type in the doc string. It's already provided in the function definition. So instead of writing  `Uint8array`, you should write what the bytes are representing.

Reply:
ok